templates/keras_cv/index.md

```diff
@@ -2,87 +2,219 @@
-
----
 ## Installation
 
-KerasCV requires **Python 3.7+** and **TensorFlow 2.9+**.
-
-Install the latest release:
+To install the latest official release:
 
 ```
-pip install keras-cv --upgrade
+pip install keras-cv tensorflow --upgrade
 ```
 
-You can also check out other versions in our
-[GitHub repository](https://github.com/keras-team/keras-cv/releases).
+To install the latest unreleased changes to the library, we recommend using
+pip to install directly from the master branch on github:
 
-## Quick Introduction
+```
+pip install git+https://github.com/keras-team/keras-cv.git tensorflow --upgrade
+```
 
-Create a preprocessing pipeline:
+## Quickstart
 
 ```python
-import keras_cv
 import tensorflow as tf
 from tensorflow import keras
+import keras_cv
 import tensorflow_datasets as tfds
 
+# Create a preprocessing pipeline with augmentations
+BATCH_SIZE = 16
+NUM_CLASSES = 3
 augmenter = keras.Sequential(
-  layers=[
-      keras_cv.layers.RandomFlip(),
-      keras_cv.layers.RandAugment(value_range=(0, 255)),
-      keras_cv.layers.CutMix(),
-      keras_cv.layers.MixUp()
+    [
+        keras_cv.layers.RandomFlip(),
+        keras_cv.layers.RandAugment(value_range=(0, 255)),
+        keras_cv.layers.CutMix(),
     ]
 )
 
-def augment_data(images, labels):
-  labels = tf.one_hot(labels, 3)
-  inputs = {"images": images, "labels": labels}
-  outputs = augmenter(inputs)
-  return outputs['images'], outputs['labels']
-```
+def preprocess_data(images, labels, augment=False):
+    labels = tf.one_hot(labels, NUM_CLASSES)
+    inputs = {"images": images, "labels": labels}
+    outputs = augmenter(inputs) if augment else inputs
+    return outputs['images'], outputs['labels']
 
-Augment a `tf.data.Dataset`:
+train_dataset, test_dataset = tfds.load(
+    'rock_paper_scissors',
+    as_supervised=True,
+    split=['train', 'test'],
+)
+train_dataset = train_dataset.batch(BATCH_SIZE).map(
+    lambda x, y: preprocess_data(x, y, augment=True),
+        num_parallel_calls=tf.data.AUTOTUNE).prefetch(
+            tf.data.AUTOTUNE)
+test_dataset = test_dataset.batch(BATCH_SIZE).map(
+    preprocess_data, num_parallel_calls=tf.data.AUTOTUNE).prefetch(
+        tf.data.AUTOTUNE)
+
+# Create a model using a pretrained backbone
+backbone = keras_cv.models.EfficientNetV2Backbone.from_preset(
+    "efficientnetv2_b0_imagenet"
+)
+model = keras_cv.models.ImageClassifier(
+    backbone=backbone,
+    num_classes=NUM_CLASSES,
+    activation="softmax",
+)
+model.compile(
+    loss='categorical_crossentropy',
+    optimizer=keras.optimizers.Adam(learning_rate=1e-5),
+    metrics=['accuracy']
+)
 
-```python
-dataset = tfds.load('rock_paper_scissors', as_supervised=True, split='train')
-dataset = dataset.batch(64)
-dataset = dataset.map(augment_data, num_parallel_calls=tf.data.AUTOTUNE)
+# Train your model
+model.fit(
+    train_dataset,
+    validation_data=test_dataset,
+    epochs=8,
+)
 ```
 
-Create a model:
+## Contributors
```

Comment:
Let's remove all the new sections below before "citing KerasCV". They are mainly targeted at repo users, not for a general audience.

Actually why don't we keep the "Disclaimer" section since that is generally appropriate.

Reply:
Done.